keyboards/2_milk/info.json

```
@@ -2,7 +2,7 @@
   "keyboard_name": "2% Milk",
   "keyboard_folder": "2_milk",
   "url": "",
-  "maintainer": "rionlion100",
+  "maintainer": "Spaceman",
```

This key is intended to reference the maintainer's GitHub username.

```suggestion
  "maintainer": "rionlion100 (Spaceman)",
```